packages/babel-preset-expo/src/__tests__/hermes-bytecode.test.ts

```diff
@@ -220,8 +220,11 @@ const LANGUAGE_SAMPLES: {
         throw 0;
       } catch {
       }`,
-    getCompiledCode() {
-      return `try{throw 0;}catch{}`;
+    getCompiledCode({ platform }) {
+      if (platform === 'web') {
+        return `try{throw 0;}catch{}`;
+      }
+      return `try{throw 0;}catch(_unused){}`;
```

Comment:
@tmikov does hermes no longer support `try{throw 0;}catch{}` without the `(_unused)` element?

Reply:
try to bisect in depth and find it's coming from @react-native/babel-preset, https://github.com/facebook/react-native/pull/52651.
originally we don't run through regenerator and now it does. (we pass `isDev=true` in the test case)